landing-pages/site/content/en/blog/implementing-stable-API-for-Apache-Airflow.md

```diff
@@ -0,0 +1,90 @@
+This was because the ``connexion`` library we were using to build the REST API was not validating
+date-time format in OpenAPI 3.0 specification. This I would later find out, was intentional.
+Connexion dropped `strict-rfc3339` because of the later license which is not compatible with
+Apache 2.0 license.
+
+I implemented a workaround on this, by defining a function called `conn_parse_datetime` in the
+API utils module. This was later refactored and thankfully, [Kamil](https://github.com/mik-laj)
+ implemented a decorator that allowed us to have cleaner code on the views while using this function.
+
+We later tried using `rfc3339-validator` whose license is compatible with Apache 2.0 licence but
+ later discarded this because with our custom date parser we were able to use duration and
+ not just date times.
+
+### Other Endpoints
+I implemented some different other endpoints. One peculiar issue I faced was because of Marshmallow 2
+not giving error when extra fields are in the request body. I implemented a `validate_unknown`
+method on the schema to handle this. Thankfully, Flask-AppBuilder updated to using Marshmallow 3,
+we quickly updated Flask-AppBuilder in Apache Airflow and started using Marshmallow 3 too.
+
+Here are some PRs I contributed that are related to the REST API:
+
+ 1. [Add event log endpoints](https://github.com/apache/airflow/pull/9227)
+    The event log would help users get information on operations performed at the UI
+
+ 2. [Add CRUD endpoints for connection](https://github.com/apache/airflow/pull/9266)
+    This PR performs DELETE, PATCH and POST operations on ``Connection``
+
+ 3. [Add log endpoint](https://github.com/apache/airflow/pull/9331)
+    This PR enable users to get Task Instances log entries
+
+ 4. [Move limit & offset to kwargs in views plus work on a configurable maximum limit](https://github.com/apache/airflow/pull/9431)
+    This helped us in having a neat code on the views and added configurable maximum limit on query results.
+
+ 5. [Update FlaskAppBuilder to v3](https://github.com/apache/airflow/pull/9648)
+    This enabled Airflow to start using v3 of Flask App Builder and also enabled the API to use
+     a modern database serializer/deserializer
+
+ 6. [Add migration guide from the experimental REST API to the stable REST API](https://github.com/apache/airflow/pull/9771)
+    This would enable users to start using the stable REST API in less time.
+
+### Follow-Ups
+There is still lots of works to be done on the REST API including writing helpful documentation.
+I still follow up on these and hopefully, we will complete the REST API before the internship ends.
+
+I am very grateful to my mentors, [Jarek](https://github.com/potiuk) and [Kaxil](https://github.com/kaxil) for their
+patience with me and for surviving my never-ending questions.
+
+Thank you so much, [Leah E. Cole](https://github.com/leahecole), for your wonderful reviews.
+
+Thanks for reading...
```

Comment:
```suggestion
Thanks for reading! 
```
You should be proud of this blog post because it's great! There's no need to use ellipsis 😸